### PR TITLE
Enable ruff to format code in docstrings

### PR DESCRIPTION
**Pull request by Armavica**
## Related Issue

- [ ] Closes #
- [ ] Related to #

## Checklist

- [x] Checked that [the pre-commit linting/style checks pass](https://docs.pymc.io/en/latest/contributing/python_style.html)
- [ ] Included tests that prove the fix is effective or that the new feature works
- [ ] Added necessary documentation (docstrings and/or example notebooks)
- [x] If you are a pro: each commit corresponds to a [relevant logical change](https://wiki.openstack.org/wiki/GitCommitMessages#Structural_split_of_changes)

## Type of change

- [ ] New feature / enhancement
- [ ] Bug fix
- [x] Documentation
- [ ] Maintenance
- [ ] Other (please specify):

----
📚 Documentation preview 📚: https://pymc--7505.org.readthedocs.build/en/7505/

### CODE REVIEW

File: pymc/distributions/multivariate.py [hunk 1]
```diff
@@ -2418,23 +2413,18 @@ class ICAR(Continuous):
         # 4x4 adjacency matrix
         # arranged in a square lattice
 
-        W = np.array([
-            [0,1,0,1],
-            [1,0,1,0],
-            [0,1,0,1],
-            [1,0,1,0]
-        ])
+        W = np.array([[0, 1, 0, 1], [1, 0, 1, 0], [0, 1, 0, 1], [1, 0, 1, 0]])
```
Comment (twiecki) [inline pymc/distributions/multivariate.py]: I feel this looked better before.

Comment (Armavica) [reply pymc/distributions/multivariate.py]: I agree, fixed.

**Review by twiecki — APPROVED** [review]

File: pymc/backends/__init__.py [hunk 2]
```diff
@@ -32,18 +32,18 @@
...
 
-    >>> trace['x', 1000:]
+    >>> trace["x", 1000:]
 
 The `get_values` method offers more control over which values are
 returned. The call below will discard the first 1000 iterations
 from each chain and keep the values for each chain as separate arrays.
 
-    >>> trace.get_values('x', burn=1000, combine=False)
+    >>> trace.get_values("x", burn=1000, combine=False)
```
Comment (twiecki) [inline pymc/backends/__init__.py:41]: Is this still called burn?

File: pymc/distributions/multivariate.py [hunk 3]
```diff
@@ -230,34 +230,30 @@ class MvNormal(Continuous):
...
     Most of the time it is preferable to specify the cholesky
     factor of the covariance instead. For example, we could
     fit a multivariate outcome like this (see the docstring
     of `LKJCholeskyCov` for more information about this)::
 
         mu = np.zeros(3)
-        true_cov = np.array([[1.0, 0.5, 0.1],
-                             [0.5, 2.0, 0.2],
-                             [0.1, 0.2, 1.0]])
+        true_cov = np.array([[1.0, 0.5, 0.1], [0.5, 2.0, 0.2], [0.1, 0.2, 1.0]])
```
Comment (twiecki) [inline pymc/distributions/multivariate.py]: also better before

File: pymc/backends/__init__.py (hunk 2 above)
Comment (Armavica) [reply pymc/backends/__init__.py:41]: It looks like it yes

https://github.com/pymc-devs/pymc/blob/79fafb04bd5e26bf1e7057e80860107aa5939c1a/pymc/backends/base.py#L438-L441

**Review by twiecki — APPROVED** [review]

**Pull request merged**